src/library/Table/TableBody.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '../styles';
+
+type Props = {
+  /** Rendered content must be TableRow */
+  children: React$Node
+};
+
+const Root = createStyledComponent(
+  'tbody',
+  {
+    color: 'inherit'
```

Comment:
is this for an "inert style"? I don't think we need those anymore with our upgrade to emotion

Reply:
Yep. Glad to know we don't need to do that anymore!